### PR TITLE
[aws][feat] CIS 1.5 report: Section 1 Access Management and Section 2 Storage

### PR DESCRIPTION
**Pull request by aquamatthias**
# Description

# To-Dos

- [x] Add test coverage for new or updated functionality
- [x] Lint and test with `tox`

# Code of Conduct

By submitting this pull request, I agree to follow the [code of conduct](https://resoto.com/code-of-conduct).

### CODE REVIEW

File: plugins/aws/resoto_plugin_aws/resource/iam.py [hunk 1]
```diff
@@ -72,8 +74,8 @@ class AwsIamAttachedPermissionsBoundary:
 @define(eq=False, slots=False)
 class AwsIamRoleLastUsed:
     kind: ClassVar[str] = "aws_iam_role_last_used"
-    mapping: ClassVar[Dict[str, Bender]] = {"last_used_date": S("LastUsedDate"), "region": S("Region")}
-    last_used_date: Optional[datetime] = field(default=None)
+    mapping: ClassVar[Dict[str, Bender]] = {"last_used": S("LastUsedDate"), "region": S("Region")}
+    last_used: Optional[datetime] = field(default=None)
```
Comment (lloesche) [inline plugins/aws/resoto_plugin_aws/resource/iam.py:78]: Should this be `atime`?

File: plugins/aws/resoto_plugin_aws/resource/iam.py [hunk 2]
```diff
@@ -393,11 +395,13 @@ def called_mutator_apis(cls) -> List[AwsApiSpec]:
 class AwsIamAccessKeyLastUsed:
     kind: ClassVar[str] = "aws_iam_access_key_last_used"
     mapping: ClassVar[Dict[str, Bender]] = {
-        "last_used_date": S("LastUsedDate"),
+        "last_used": S("LastUsedDate"),
```
Comment (lloesche) [inline plugins/aws/resoto_plugin_aws/resource/iam.py:398]: This sounds like `atime` as well.

**Review by lloesche — DISMISSED** [review]

File: plugins/aws/resoto_plugin_aws/resource/iam.py (hunk 1 above)
Comment (aquamatthias) [reply plugins/aws/resoto_plugin_aws/resource/iam.py:78]: already defined as such.

File: plugins/aws/resoto_plugin_aws/resource/iam.py (hunk 2 above)
Comment (aquamatthias) [reply plugins/aws/resoto_plugin_aws/resource/iam.py:398]: Added for root user.

**Review by lloesche — APPROVED** [review]

**Pull request merged**